### PR TITLE
Osciloscope gimun

### PR DESCRIPTION
**Pull request by mrgomdev**
Solved osciloscope.
The limits of problem are written at the first of the code.

Comments will be added more.

### CODE REVIEW

**Comment by mrgomdev** [comment]
@gichani

**Review by simpleboss — COMMENTED** [review]
line 26, 와즈디스?
line 57, 와즈디스?

totally, i don't understand.

**Comment by mrgomdev** [comment]
Those two lines are comments

**Comment by simpleboss** [comment]
plz explain the

**Comment by mrgomdev** [comment]
http://www.letmegooglethat.com/?q=string+multiply+python

File: .gitignore [hunk 1]
```diff
@@ -22,6 +22,9 @@ bld/
 [Oo]bj/
```
Comment (simpleboss) [inline .gitignore]: https://www.google.co.kr/search?q=Oo+bj%2F&oq=Oo+bj%2F&aqs=chrome..69i57j0l2.8442j0j4&sourceid=chrome&ie=UTF-8

'No useful results'
What's this?

File: .gitignore [hunk 2]
```diff
@@ -22,6 +22,9 @@ bld/
...
 [Ll]og/
```
Comment (simpleboss) [inline .gitignore]: What's this?

**Review by simpleboss — COMMENTED** [review]
Oh. this is .gitignore file

File: 4MultipleLoop/osciloscope_gimun/osciloscope.py [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+
+
+def print_chunks(amplitude, line_length=MAX_LINE_LENGTH):
+    for y in range(amplitude, -(amplitude + 1), -1):
+        print build_line(amplitude, y, line_length) + '|{:3}V'.format(y)
+
+
+def main(amplitude, line_length):
+    def postfix(y):
+        return '|{:3}V'.format(y)
```
Comment (simpleboss) [inline 4MultipleLoop/osciloscope_gimun/osciloscope.py:57]: i don't know format.
http://infohost.nmt.edu/~shipman/doc/python27/web/new-str-format.html
I'm reading this page

Comment (simpleboss) [reply 4MultipleLoop/osciloscope_gimun/osciloscope.py:57]: lots of function in format.  helpful

File: 4MultipleLoop/osciloscope_gimun/osciloscope.py [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+    return line
+
+
+def print_chunks(amplitude, line_length=MAX_LINE_LENGTH):
+    for y in range(amplitude, -(amplitude + 1), -1):
+        print build_line(amplitude, y, line_length) + '|{:3}V'.format(y)
+
+
+def main(amplitude, line_length):
+    def postfix(y):
```
Comment (simpleboss) [inline 4MultipleLoop/osciloscope_gimun/osciloscope.py:56]: why def in def? is it necessary?

File: 4MultipleLoop/osciloscope_gimun/osciloscope.py [hunk 5]
```diff
@@ -0,0 +1,88 @@
...
+        print line + postfix(y)
+
+
+def test(amplitude, line_length):
+    # print_chunks(n, line_length)
+    main(amplitude, line_length)
+
+
+if __name__ == '__main__':
+    n = input()
```
Comment (simpleboss) [inline 4MultipleLoop/osciloscope_gimun/osciloscope.py:83]: n could be amplitude. why n?

File: 4MultipleLoop/osciloscope_gimun/osciloscope.py [hunk 6]
```diff
@@ -0,0 +1,88 @@
...
+    n = int(raw_input())
+    if n < 1:
+        raise ValueError('input amplitude is too small to draw')
+    if n >= MAX_LINE_LENGTH:
+        raise ValueError('input amplitude is larger than MAX_LINE_LENGTH')
+    return n
+
+
+def build_line(amplitude, y, max_x):
+    def build_span(len_total, len_left_blanks):
```
Comment (simpleboss) [inline 4MultipleLoop/osciloscope_gimun/osciloscope.py:15]: before def build_span, no assignment len_left_blanks.

**Pull request closed without merging**